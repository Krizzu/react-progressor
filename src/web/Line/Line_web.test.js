const add = (a, b) => a + b;

// testing
test('Add 1 + 2 to equal 3', () => {
  expect(add(1, 2)).toBe(3);
});
