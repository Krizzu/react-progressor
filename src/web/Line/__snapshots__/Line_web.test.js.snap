// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders without errors 1`] = `
<div
  style={
    Object {
      "height": 5,
      "position": "relative",
      "width": 150,
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#eee",
        "height": 5,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": 150,
      }
    }
  />
  <div
    style={
      Object {
        "backgroundColor": "#8DE969",
        "height": 5,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "transition": "width 200ms linear",
        "width": 0,
      }
    }
  />
</div>
`;
